### ISSUE TITLE
Feedback for “Introduction”

### ISSUE THREAD

**Issue by lechmigdal** [dair-ai/Prompt-Engineering-Guide#187]
OpenAI playground has currently default of temperature=1.0 (not 0.7 as suggested in the text). At least this is how it opens for me all the time.

**Comment by omarsar** [comment]
@lechmigdal Good point. I am revising the content and will update this accordingly.
